GazePianoPrototype/KeySelector.xaml

```diff
@@ -27,31 +28,31 @@
         
-        <Button Grid.Row="1" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="A" Click="Button_Click"/>
-        <Button Grid.Row="1" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="A#" Click="Button_Click"/>
-        <Button Grid.Row="1" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Ab" Click="Button_Click"/>
+        <Button Grid.Row="1" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="A" Tag="A" Click="Button_Click"/>
+        <Button Grid.Row="1" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="A&#x0266F;" Tag="A#" Click="Button_Click"/>
+        <Button Grid.Row="1" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="A&#x0266D;" Tag="Ab" Click="Button_Click"/>
 
-        <Button Grid.Row="2" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="B" Click="Button_Click"/>
-        <Button Grid.Row="2" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="Bb" Click="Button_Click"/>
+        <Button Grid.Row="2" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="B" Tag="B" Click="Button_Click"/>
+        <Button Grid.Row="2" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="B&#x0266D;" Tag="Bb" Click="Button_Click"/>
 
-        <Button Grid.Row="3" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="C" Click="Button_Click"/>
-        <Button Grid.Row="3" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="C#" Click="Button_Click"/>
-        <Button Grid.Row="3" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Cb" Click="Button_Click"/>
+        <Button Grid.Row="3" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="C" Tag="C" Click="Button_Click"/>
+        <Button Grid.Row="3" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="C&#x0266F;" Tag="C#" Click="Button_Click"/>
+        <Button Grid.Row="3" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="C&#x0266D;" Tag="Cb" Click="Button_Click"/>
 
-        <Button Grid.Row="4" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="D" Click="Button_Click"/>
-        <Button Grid.Row="4" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="D#" Click="Button_Click"/>
-        <Button Grid.Row="4" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Db" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="D" Tag="D" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="D&#x0266F;" Tag="D#" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Db" Tag="Db" Click="Button_Click"/>
```

Comment:
The b should be swapped to a flat symbol as well

Reply:
Thanks for catching that. Okay it's resolved now.

Reply:
Awesome. Just waiting for the build to complete and then I'll complete the pr.